recipes-server/grafana/grafana.bb

```diff
@@ -74,7 +74,7 @@ do_install() {
     cp ${D}${bindir}/grafana-server ${D}${datadir}/grafana/bin/grafana-server
 }
 
-INSANE_SKIP:${PN} = "ldflags already-stripped build-deps"
+INSANE_SKIP:${PN} += "ldflags already-stripped build-deps"
```

Comment:
Can we squash to first commit?

Reply:
It is too late to squash a part of mid-places commit into the first one, I'm afraid